docs/how-to-connect-gpt-extensions.md

```diff
@@ -4,151 +4,159 @@ First to integrate this extension, you need to use the following versions of the
 
     @gravity-ui/markdown-editor version 13.18.0 or higher
 
-### 1. Add extension usage to extension builder 
 
-```ts
-    import {gptExtension} from '@gravity-ui/markdown-editor';
+Features:
 
-    builder.use(
-        gptExtension,
-        // The next step add gptWidgetProps,
-    )          
-```
-### 2. Add gpt extensions props
+<img src="./assets/gifs/custom-prompt-preset-gpt.gif" width="470"/>
 
-```ts
-    import {gptExtension, type GptWidgetOptions} from '@gravity-ui/markdown-editor';
+<img src="./assets/gifs/prompt-preset-gpt.gif" width="470"/>
 
-    const gptWidgetProps: GptWidgetOptions = {
-        // add params
-    }
+### 1. Add extension usage and extensions props
 
-    builder.use(
-        gptExtension,
-        gptWidgetProps, // use params
-    )        
-```
+```ts
+import React from 'react';
 
-#### Example of implementation
+import {
+    gptExtension,
+    MarkdownEditorView,
+    useMarkdownEditor,
+} from '@gravity-ui/markdown-editor';
 
-    setYfmRaw - a setter to change the text in an editor
+export const Editor: React.FC<EditorProps> = (props) => {
+    const mdEditor = useMarkdownEditor({
+        // ...
+        extraExtensions: (builder) =>
+            builder.use(
+                ...
+                // add gpt extension
```

Comment:
use uppercase in comment in text for `GPT`
https://en.wikipedia.org/wiki/Generative_pre-trained_transformer

Reply:
in README.md too